gnmawsgr/tasks.py

```diff
@@ -276,12 +276,7 @@ def do_glacier_restore(request_id,itemid,path):
     while True:
         try:
             with open(filename,'wb') as fp:
-                try:
-                    key.get_file(fp, cb=partial(download_callback, rq), num_cb=40)
-                except AttributeError:
-                    raven_client.captureException()
-                    logger.error("No Key found in S3. Object values: {0}".format(rq.__dict__))
-                    return
```

Comment:
Though if that was done, it would be a good idea to write the logger line like this: -

logger.error("Attribute Error Object values: {0}".format(rq.__dict__))

Because written like that an attribute error on another line might cause it to execute.